components/ui/BannerTextButton.tsx

```diff
@@ -0,0 +1,79 @@
+import { Picture, Source } from "deco-sites/std/components/Picture.tsx";
+import type { Image as LiveImage } from "deco-sites/std/components/types.ts";
+
+export interface BannerTextButton {
+  /** @description text to be rendered on top of the text */
+  title?: string;
+  /** @description Text to be rendered after the title */
+  text: string;
+  /** @description href where the button will send to */
+  href?: string;
+  /** @description justify and align the text (MOBILE ONLY)*/
+  textAlign?: "center" | "left";
```

Comment:
Deixa ele alinhar com o mesmo valor em ambos
```suggestion
  /** @description justify and align the text*/
  textAlign?: "center" | "left";
```